src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -128,6 +128,8 @@ public interface IPreferences
         /// </summary>
         bool ShowEdges { get; set; }
 
+        List<string> NamespacesToExcludeFromLibrary { get; set; }
```

Comment:
I think this is a binary compatibility concern. i.e. old implementers of this interface no longer implement it.